spylib/oauth/exchange_token.py

```diff
@@ -0,0 +1,114 @@
+from typing import Union
+
+from httpx import AsyncClient, codes
+
+from .types import OfflineTokenModel, OnlineTokenModel
+
+
+async def exchange_token(
+    *,
+    shop: str,
```

Comment:
I expect that would cause more harm than good - especially when vscode is trying to pick up autocomplete or we are trying to generate docs. No perfect solution I'm afraid

Reply:
I'm using shop because that is consistent with all of the Shopify api calls (auth, session, webhook, etc.)

I guess I feel like we'd need a very good reason to change it?

Reply:
ok. I see what you mean. ref: https://shopify.dev/apps/auth/oauth/getting-started#step-4-confirm-installation